Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/InteractableToggleCollection.cs

```diff
@@ -47,27 +91,35 @@ public int CurrentIndex
 
         private List<UnityAction> toggleActions = new List<UnityAction>();
 
+        private bool listenersAdded = false;
```

Comment:
Instead of using listenersAdded, I believe you can just add the init in on Awake(). If people modify the ToggleList in their Start() or some other component, they'll be covered.

Reply:
If Awake is used instead of Start, our current TestRadialSetPrefab test in InteractableTests fails.

Reply:
Differrent suggestion, to remove listenersAdded, use the equivalent 
listenersAdded == !isEmpty(toggleActions)
so in start just check if toggleActions.count > 0

Reply:
I like that suggestion, adding it now.